src/data/markdown/translated-guides/en/02 Using k6/05 k6 Options/02 Reference.md

```diff
@@ -1425,6 +1425,26 @@ export const options = {
 
 
 
+## Upload Only
+
+A boolean specifying whether the test should just be uploaded to the cloud, but not run it. Available in `k6 cloud` command.
+
+This is useful if you want to just update a given test but to run it later. For example to update it from the CI of the test, but run the test on a schedule.
```

Comment:
Maybe something like that?

```suggestion
This would be useful if you would like to update a given test and run it later. For example, updating test scripts of a scheduled test from the CI pipelines.
```